src/actionHandlers.ts

```diff
@@ -12,12 +12,12 @@ export type ActionHandler<C, S, P> = (params: {
 }) => void
 
 export type ActionHandlerSet<C, S> = {
-  [key: number]: ActionHandler<C, S, any>[]
+  [key: string]: ActionHandler<C, S, any>[]
```

Comment:
This is the JS object (think hashmap) that maps action names to their "handlers" (callback functions that run when this action triggers). It used to be an int (from the enum). Now it is a string.